### PR TITLE
Fix: #disqus_tread.on_shown was always picking out the first accordion

### LINKED ISSUE #157: Bug: disqus_scripts grabs the first accordion element, not necessarily the correct one
On line 27 [here](https://github.com/talha131/pelican-elegant/blob/master/templates/_includes/disqus_scripts.html#L27), the script grabs all elements of class 'accordion-toggle' and then proceeds to modify the first one of them. 

If there are more accordions than the one from the comment section, this will modify the wrong one.

### PR DESCRIPTION
**Pull request by leotrs**
If there were more accordions in the page, this would not have changed the correct one. Now refers to it by the newly added id. Fixes #157.

### CODE REVIEW

**Comment by iranzo** [comment]
Problem seems that you commited more changes to 'master' in the meantime and this got under a pile of other change son top.

Thanks

**Comment by iranzo** [comment]
@silverhook assign this to me too please

File: templates/_includes/disqus_scripts.html [hunk 1]
```diff
@@ -24,13 +24,13 @@
...
-    var old_innerHTML = link[0].innerHTML;
-    $(link[0]).fadeOut(500, function() {
+    var link = document.getElementById('disqus-accordion-toggle');
+    var old_innerHTML = link.innerHTML;
+    $(link).fadeOut(500, function() {
         $(this).text('Click here to hide comments').fadeIn(500);
     });
     $('#disqus_thread').on('hidden', function () {
-        $(link[0]).fadeOut(500, function() {
+        $(link).fadeOut(500, function() {
```
Comment (iranzo) [inline templates/_includes/disqus_scripts.html]: This is the change I've tested and that should be on an independent PR branch

**Comment by iranzo** [comment]
This will also require the id to be used being the right one, on my rendered website it still shows div with name accordion-toggle, so that probably needs to be put inside.

( templates/_includes/comments.html right? )

**Comment by silverhook** [comment]
@leotrs, it seems you don’t use Elegant on your blog right now. Can you at least indicate whether you have interest in pushing this pull request forward still regardless?

Some relevant info: Elegant has grown now from a single-maintainer to a community-led project, so we’re picking up speed at an astounding pace (see #173 for more info). It might be the perfect time to join in, if Elegant is still on your radar ;)

**Comment by leotrs** [comment]
@silverhook I use it [here](www.erdosnet.work). Apologies for the mess here. Please advise on how to proceed with this PR, or else feel free to close this and submit the changes yourself. I'm glad someone is taking the reins! I :heart: Elegant.

**Comment by silverhook** [comment]
@leotrs, great to have you on board! :smile: 

@iranzo volunteered to be the reviewer responsible for this PR, so it would be best to sync with him regarding how to clean up this PR and get it merged.

**Comment by iranzo** [comment]
@leotrs Hola!
As this was against your master branch now it is a bit messy, could it be possible for you to create a new PR with the changes relevant for disqus accordion on a different one so that we can close this when that one gets created and merged?

Thanks!
Pablo

**Comment by talha131** [comment]
Close via 484b0ee1b97e568393fedf21f33ed51817341f01
